soc/xtensa/intel_adsp/common/include/cavs-mem.h

```diff
@@ -23,17 +23,6 @@
 #define HP_SRAM_WIN3_BASE (L2_SRAM_BASE + CONFIG_ADSP_WIN3_OFFSET)
 #define HP_SRAM_WIN3_SIZE 0x2000
 
-#define IMR_BOOT_LDR_MANIFEST_BASE CONFIG_IMR_MANIFEST_ADDR
-#define IMR_BOOT_LDR_TEXT_ENTRY_BASE (IMR_BOOT_LDR_MANIFEST_BASE + 0x6000)
-
-/* These are fake section addresses used only in the linker scripts */
-#define IDT_BASE		(L2_SRAM_BASE + L2_SRAM_SIZE)
-#define IDT_SIZE		0x2000
-#define UUID_ENTRY_ELF_BASE	0x1FFFA000
-#define UUID_ENTRY_ELF_SIZE	0x6000
-#define LOG_ENTRY_ELF_BASE	0x20000000
-#define LOG_ENTRY_ELF_SIZE	0x2000000
-#define EXT_MANIFEST_ELF_BASE	(LOG_ENTRY_ELF_BASE + LOG_ENTRY_ELF_SIZE)
-#define EXT_MANIFEST_ELF_SIZE	0x2000000
+#define IMR_BOOT_LDR_TEXT_ENTRY_BASE (CONFIG_IMR_MANIFEST_ADDR + 0x6000)
```

Comment:
What is the 0x6000 here do we know? Can we define it to make it easier to understand?

Reply:
Rimage places the start of the "TEXT" section it parsed out of the bootloader module 24k after the location of the manifest in the output file, which is itself a rimage-generated data structure.  It's just the way it works, basically.  I can move the magic number around or give it different names but it doesn't stop being magic.  And it lives at exactly one place in the code now.

Reply:
Fair enough, the rest of this is so well commented this little magic num seems out of place given the lack of commentary, the blurb you've put here would be helpful in code to better understand it the next time (inevitably) someone has to read it

Reply:
Agreed, I'll see what I can do.  Really this is sort of a wart in rimage.  We should just say "put our boot blob at this address" and it should figure out the padding and give a single yea/nay answer as to whether that fits with the hardware requirements (actually it's CSME software that defines the image format, so ultimately the offsets get designed there, way outside of the world of open source visibility).